curvefs/src/metaserver/storage/rocksdb_storage.h

```diff
@@ -183,10 +183,10 @@ class RocksDBStorage : public KVStorage, public StorageTransaction {
     void InitDbOptions();
 
  private:
-    bool inited_;
+    bool inited_ = false;
```

Comment:
why init these members in declare?

Reply:
just give them a default value, because if you don't initialize them in constructor, the value is unspecified, and some assert will fail.